.editorconfig

```diff
@@ -233,32 +231,34 @@ dotnet_naming_style.first_upper.capitalization = first_word_upper
 # prefix_interface_interface_with_i - Interfaces must be PascalCase and the first character of an interface must be an 'I'
 dotnet_naming_style.prefix_interface_interface_with_i.capitalization = pascal_case
 dotnet_naming_style.prefix_interface_interface_with_i.required_prefix = I
+# disallowed_by_design_guidelines - use to auto-format items that are disallowed by the design guidelines
+dotnet_naming_style.prefix_with_underscore.capitalization = pascal_case
+dotnet_naming_style.prefix_with_underscore.required_prefix = _
```

Comment:
This `.editorconfig` tries to match the default StyleCop rules which uses `this.` instead of `_`. I realize that this splits the C# community in to two camps and is controversial.